core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -266,15 +233,64 @@ static AggregationData createAggregationData(MutableAggregation aggregation) {
         CreateMeanData.INSTANCE,
         CreateStdDevData.INSTANCE);
   }
-  
+
+  // Covert a mapping from TagValues to MutableAggregation, to a mapping from TagValues to
+  // AggregationData.
+  private static Map<List<Object>, List<AggregationData>> createAggregationMap(
+      Map<List<Object>, List<MutableAggregation>> tagValueAggregationMap) {
```

Comment:
Suggestion: You could change the signature to

```java
  private static <T> Map<T, List<AggregationData>> createAggregationMap(
      Map<T, List<MutableAggregation>> tagValueAggregationMap) {
```

to show that the key type isn't relevant.

Reply:
Sure, updated.